applications/error-notifications.md

```diff
@@ -0,0 +1,32 @@
+# Error Notifications
+
+In order to alert teams to server errors for applications we need a way of
+posting these out from applications.  Currently we have development and
+production instances of [Errbit](https://github.com/errbit/errbit), an
+open source product compatible with [airbrake](https://airbrake.io/)
+
+Details of these instances can be obtained from your local development lead.
+
+## NodeJS
+
+[node-airbrake](https://github.com/airbrake/node-airbrake) can be used for Node
+applications.  It has recently had an update for compatibility with HapiJS
+(it already had expressJS), however in order to load it correctly as a plugin
+or using glue and a manifest please see the HapiJS section below.
+
+### HapiJS
```

Comment:
Not clear on the question... But if using node but not hapi, ie with express then the vanilla node-airbrake will suffice.  You could use node-airbrake with hapijs, but it's not in the traditional plugin way, the node-hapi-airbrake wraps it with the plugin stuff native to hapi. it also exposes a couple of variables to manipulate for ease of use.